public/test/examples/bundle.js

```diff
@@ -116,7 +116,7 @@ function update(obj, arr) {
   arr.push("new");
 }
 function recordingFinished() {
-  dump(`RecReplaySendAsyncMessage Example__Finished`);
+  window.Example__Finished = true; dump(`RecReplaySendAsyncMessage Example__Finished`);
```

Comment:
This seems like it should be centralized someplace to avoid the shotgun surgery.